package.json

```diff
@@ -10,18 +10,25 @@
     "bundle": "node node_modules/react-360/scripts/bundle.js",
     "open": "node -e \"require('xopen')('http://localhost:8081/index.html')\"",
     "devtools": "react-devtools",
-    "test": "jest"
+    "lint": "eslint ./src"
```

Comment:
Maybe you can be more precise with the file you would analyse, you could add something like `src/**/*.js src/**/*.jsx`